web/thesauruses/c/17/operators.json

```diff
@@ -0,0 +1,202 @@
+{
+  "meta": {
+    "language": "c",
+    "language_version": "17",
+    "language_name": "C",
+    "structure": "operators"
+	},
+  "concepts": {
+    "addition": {
+      "code": "+",
```

Comment:
For the operators, can you actually place where the operands are? (like C would be `a + b` but Lisp has `(+ a b)`. The order of operands and operators matter!)